tests/js/spec/views/accountSecurity.spec.jsx

```diff
@@ -20,13 +21,18 @@ describe('AccountSecurity', function() {
     let wrapper = shallow(<AccountSecurity />, TestStubs.routerContext());
 
     expect(wrapper.find('EmptyMessage')).toHaveLength(1);
+    expect(wrapper.find('TwoFactorRequired')).toHaveLength(0);
   });
 
   it('renders a primary interface that is enrolled', function() {
     Client.addMockResponse({
       url: ENDPOINT,
       body: [TestStubs.Authenticators().Totp({configureButton: 'Info'})],
     });
+    Client.addMockResponse({
```

Comment:
I think we can just move this into the `beforeEach` (or even a `beforeAll` which runs just once  inside of a `describe` block).